NeuroID/TrackerEvents/AppEvents.swift

```diff
@@ -30,4 +33,9 @@ internal extension NeuroIDTracker {
     @objc func appMovedToForeground() {
         captureEvent(event: NIDEvent(type: NIDEventName.windowFocus))
     }
+    
+    @objc func appLowMemoryWarning() {
+        setLowMemory(true)
```

Comment:
This isn't an extension of NeuroID but NeuroID Tracker so I think you need to call NeuroID.isLowMemory